### PR TITLE
lib/core/bytes: add `Int::to_bytes` and `Bytes::to_i`

### PR DESCRIPTION
**Pull request by privat**
More nice services on bytes.

### CODE REVIEW

**Comment by privat** [comment]
dedicated to @TheBaconMan

**Comment by privat** [comment]
ping @xymus or @R4PaSs ?

File: lib/core/bytes.nit [hunk 1]
```diff
@@ -613,6 +651,58 @@ private class BytesIterator
 	redef fun item do return tgt[index]
 end
 
+redef class Int
+	# Returns the big-endian representation of self.
```
Comment (Morriar) [inline lib/core/bytes.nit]: I don't understand why is this `big-endian` without parenthesis when the previous comment states:

> Returns self interpreted as a (big-endian) positive integer.

Is the endianess more important in the second method? Am  I missing something?

**Comment by Morriar** [comment]
+1

File: lib/core/bytes.nit (hunk 1 above)
Comment (privat) [inline lib/core/bytes.nit]: It's just a lack of consistency. I will remove the first pair of parenthesis

**Comment by privat** [comment]
update: improve doc

**Comment by R4PaSs** [comment]
:shipit:

**Pull request merged**